samples/csharp/chatapp/SampleChatHub.cs

```diff
@@ -0,0 +1,38 @@
+﻿using Azure.Messaging.WebPubSub;
+using Microsoft.Azure.WebPubSub.AspNetCore;
+using Microsoft.Azure.WebPubSub.Common;
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace chatapp
+{
+    public class SampleChatHub : WebPubSubHub
+    {
+        private readonly WebPubSubServiceClient _serviceClient;
+
+        public SampleChatHub(WebPubSubServiceClient serviceClient)
+        {
+            _serviceClient = serviceClient;
+        }
+
+        public override ValueTask<WebPubSubEventResponse> OnConnectAsync(ConnectEventRequest request, CancellationToken cancellationToken)
+        {
+            // not register event will never be triggered.
+            throw new NotImplementedException();
+        }
+
+        public override async Task OnConnectedAsync(ConnectedEventRequest request)
+        {
+            await _serviceClient.SendToAllAsync($"[SYSTEM] {request.ConnectionContext.UserId} joined.");
+        }
+
+        public override async ValueTask<WebPubSubEventResponse> OnMessageReceivedAsync(UserEventRequest request, CancellationToken cancellationToken)
```

Comment:
return base.OnMessageReceivedAsync(request, cancellationToken);

Reply:
`base` is abstract. Then use "ack" as sample.